index.html

```diff
@@ -61,6 +61,324 @@ <h1 class="display-5 fw-bold primary-text">I’m Yannick<br> Glad to see you!</h
+                 </p>
+
+                 <div class="work-tech">
+                 <ul>
+                   <li>html</li>
+                   <li>Ruby on Rails</li>
+                   <li>css</li>
+                   <li>javascript</li>
+                 </ul>
+                 </div>
+                <div>
+              </div>
+              </div>
+              
+              <button class="btn btn-one" type="button">See Project</button>
+            
+         </div>
+          
+          <div class="flex-item-right work3">
+              
+               <img src="image/portfolio4.png" class="img-responsive" alt="image-work1">
+ 
+          </div>
+        </div>
+     
+     </div>
+  
+  </section>
+      
+    
+  <section class="about-myself">
+        <div class="container py-5">
+            
+            <div class="row align-items-center justify-content-center">
+                                <div class="col-lg-6">
+
+                <h1 class="display-5 fw-bold primary-text">About Myself<br></h1>
+                 <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                 <a class="btn-connect">Let's connect</a>
+                <br><br>
+
+                <div class="media">
+                    <div class=""><a href="#"><i class="fa fa-twitter"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-linkedin"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-medium"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-github"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-angellist"></i></a></div>
+                </div>
+                                    
+                <button class="btn btn-one" type="button">Get My Resume</button>
+              </div>
+            
+             <div class="col-lg-6">
+                 <div class="row">
+                     <div class="col-lg-12 col-md-12">
+                         <h2> Languages </h2>
+                         <div class="grid-container">
+                          <div>
+                              
+                              <img src="image/js.png" alt="js_logo">
+                              <p>JavaScript</p>
+                             
+                          </div>
+                          <div>
+                              
+                              <img src="image/html.png" alt="html_logo">
+                              <p>HTML</p>
+                             
+                          </div>
+                             
+                          <div>
+                              
+                              <img src="image/css.png" alt="css_logo">
+                              <p>CSS</p>
```

Comment:
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. Kindly fix this.